lighthouse-core/gather/computed/metrics/lantern-first-cpu-idle.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Node = require('../../../lib/dependency-graph/node');
+const CPUNode = require('../../../lib/dependency-graph/cpu-node'); // eslint-disable-line no-unused-vars
+const NetworkNode = require('../../../lib/dependency-graph/network-node'); // eslint-disable-line no-unused-vars
+
+const FirstInteractive = require('../first-interactive');
+const LanternConsistentlyInteractive = require('./lantern-consistently-interactive');
+
+class FirstCPUIdle extends LanternConsistentlyInteractive {
```

Comment:
> this just reuses the graphs and coefficients of TTI combined with the existing algorithm for first interactive

one downside to this approach is it's getting difficult to layer all three (four?) of these in my head to follow along :)